core-modules/generic-core-terms-03-our-commitments.md.md

```diff
@@ -0,0 +1,8 @@
+## Our Commitments
+
+When working with us, according to this policy, you can expect us to:
+
+- Respond to your report promptly, and work with you to understand and validate your report 
+- Strive to keep you informed about the progress of a vulnerability as it is processed
+- Work to remediate discovered vulnerabilities in a timely manner
+- Extend Safe Harbor for your vulnerability research that is related to this policy
```

Comment:
Presuming it falls within the bounds as explained within? Or is that implicit here?

Reply:
Great question - It's implicit